### DISCUSSION TITLE
ttk style for CTkEntry ,ComboBox and TextBox

### DISCUSSION THREAD

**Discussion by onlyforpeace** [TomSchimansky/CustomTkinter#1850] [General]
Hi!! i search to change the standard color of the entry or textbox text when state='disabled'.
in fact, I've got a form that must be read only by employes, so i use the disabled state of ctk.CTkEntry, ComboBox and TextBox, but in this statement the policy is lightgrey, so it's difficult to read it, so I would like that be for exemple in Black when the state is disabled, and Blue when the state is normal, i tried:
`style = ttk.Style()
style.configure('Appstyle', foreground='black') 
style.map('Appstyle',
    foreground=[('disabled', 'blue')])`

but style is not supported by ctkEntry... is there a way for this?
thank's alot for your help.

**Comment by avalon60** [comment]
This is not supported. Unfortunately there are some _configure_ settings which seem to have been omitted,  for certain widgets (e.g. for CTkEntry & CTkTextBox).  Also see issues #1557 / #1837. 

You can do the CTkComoBox, by using 

`widget.configure(text_color_disabled="<colour>")`

You would need to raise an issue, to get the other examples considered by @TomSchimansky.

**Reply by onlyforpeace** [reply]
Thank you so much @avalon60 for your response, but I'm not familiar with this GitHub request system as I'm still a novice in this area. Could you please explain to me a bit about how it works? I have a feeling that it's an essential topic in programming.

**Reply by avalon60** [reply]
There is an "Issues" tab at the top of the repo page:

![image](https://github.com/TomSchimansky/CustomTkinter/assets/89534395/2eb7f5ee-d167-4bbd-97d0-c2ab052301f6)

I underscored the tab in yellow.

Click on that and you raise an issue in a similar way you post a question on this tab. Just click the _New issue_ button.  I tend to start with a title which includes the CustomTkinter version. Example:

_CTk 5.2.0: CTkTabview.configure(fg_color=...) does not update the whole foreground_
